### PR TITLE
php-excimer: init at 1.2.0

### PR DESCRIPTION
**Pull request by shyim**
Excimer is a profiler made by the mediawiki foundation to trace PHP.
https://www.mediawiki.org/wiki/Excimer

Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Comment by shyim** [comment]
1.2.0 is the latest tag, but 1.2.1 is available. I asked the IRC if they could add the missing tag.

File: php-8.1-excimer.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        - php-excimer-config=${{package.full-version}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}/etc/php/conf.d"
+          echo "extension=excimer.so" > "${{targets.subpkgdir}}/etc/php/conf.d/excimer.ini"
+
+update:
+  enabled: true
+  github:
+    identifier: php-excimer/php-excimer
```
Comment (ajayk) [inline php-8.1-excimer.yaml:59]:
```suggestion
    identifier: wikimedia/mediawiki-php-excimer
```

File: php-8.2-excimer.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        - php-excimer-config=${{package.full-version}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}/etc/php/conf.d"
+          echo "extension=excimer.so" > "${{targets.subpkgdir}}/etc/php/conf.d/excimer.ini"
+
+update:
+  enabled: true
+  github:
+    identifier: php-excimer/php-excimer
```
Comment (ajayk) [inline php-8.2-excimer.yaml:59]:
```suggestion
    identifier: wikimedia/mediawiki-php-excimer
```

File: php-8.3-excimer.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        - php-excimer-config=${{package.full-version}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}/etc/php/conf.d"
+          echo "extension=excimer.so" > "${{targets.subpkgdir}}/etc/php/conf.d/excimer.ini"
+
+update:
+  enabled: true
+  github:
+    identifier: php-excimer/php-excimer
```
Comment (ajayk) [inline php-8.3-excimer.yaml:59]:
```suggestion
    identifier: wikimedia/mediawiki-php-excimer
```

**Comment by shyim** [comment]
is now up to date wolfictl update should pass :)

**Review by ajayk — APPROVED** [review]

**Pull request merged**